packages/orchestrator/internal/sandbox/uffd/serve.go

```diff
@@ -108,9 +110,8 @@ outerLoop:
 			}
 
 			if err == syscall.EAGAIN {
-				logger.Debug("uffd: eagain error, going back to polling", zap.Error(err), zap.Int("read_bytes", n))
-
 				// Continue polling the fd.
+				time.Sleep(time.Millisecond)
```

Comment:
This isn't a log line remove

Reply:
sorry, yup, this should be a different PR if it's worth doing. thanks for keeping me honest!